packages/datadog-instrumentations/src/mysql2.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const { AsyncResource } = require('async_hooks')
+const {
+  channel,
+  addHook,
+  bind,
+  bindAsyncResource,
+  bindEventEmitter
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+addHook({ name: 'mysql2', file: 'lib/connection.js', versions: ['>=1'] }, Connection => {
+  const startCh = channel('apm:mysql2:addCommand:start')
```

Comment:
Basically, this should probably match the `mysql` naming.

Reply:
It's worth checking if we can somehow supports both mysql and mysql2 in one plugin.

Reply:
Agree. At first both plugins were very similar but they ended up being different as the code of mysql2 diverged from the original library, but with the new plugin system we could normalize everything on the publisher side so that a single subscriber handles the messages the same way for both instrumentations.